documentation/versioned_docs/version-1.6.0/License and Bill of Materials

```diff
@@ -0,0 +1,17 @@
+---
+id: License and Bill of materials
+sidebar_position: 6
```

Comment:
Please rename the file and add the file ending `.md`. Otherwise docusaurus won't pick it up.